llabake/Courses-Management-PS
# Ft build application components

#### What does this PR do?
* Add application component pages

#### Description of Task to be completed?
* Set up production webpack
* Build loader 
* Build Not found page
#### How should this be manually tested?
* Clone the repo 
* Install dependencies using `npm install`
* Run `npm start` to start the application
* You should see the browser page open on `localhost:3000`
* Navigate through the app

#### What are the relevant pivotal tracker stories?
* N/A
